src/components/map-overlays/custom-marker.component.tsx

```diff
@@ -8,7 +8,7 @@ interface IProps {
   location: Location;
   onPress: () => void;
   text: string;
-  markerType: "campus" | "building" | "poi";
+  markerType: "campus" | "building" | "poi" | "destination";
```

Comment:
Is this additional type still needed?

Reply:
@osahtout This is related to the markerType comment below. Can you remove this if not needed